src/north/influxdb/InfluxDB.class.js

```diff
@@ -2,6 +2,25 @@ const { vsprintf } = require('sprintf-js')
 
 const ApiHandler = require('../ApiHandler.class')
 
+/**
+ * function return the content of value, that could be a Json object with path keys given by string value
+ * without using eval function
+ * @param {*} value - simple value (integer or float or string, ...) or Json object
+ * @param {*} pathValue - The string path of value we want to retrieve in the Json Object
+ * @return {*} The content of value depending on value type (object or simple value)
+ */
+const getJsonValueByStringPath = (value, pathValue) => {
```

Comment:
I would give an example of the output of this function.
Also, we are using a library called ""object-path" already and I feel it can provide the same result.